lib/vsc/utils/fancylogger.py

```diff
@@ -450,18 +450,24 @@ def logToFile(filename, enable=True, filehandler=None, name=None, max_bytes=MAX_
 
     if you want to disable logging to file, pass the earlier obtained filehandler
     """
-    handleropts = {'filename': filename,
-                   'mode': 'a',
-                   'maxBytes': max_bytes,
-                   'backupCount': backup_count,
-                   }
-    return _logToSomething(logging.handlers.RotatingFileHandler,
-                           handleropts,
-                           loggeroption='logtofile_%s' % filename,
-                           name=name,
-                           enable=enable,
-                           handler=filehandler,
-                           )
+    handleropts = {
+        'filename': filename,
+        'mode': 'a',
+        'maxBytes': max_bytes,
+        'backupCount': backup_count,
+    }
+    # logging to a file is going to create the file later on, so let's try to be helpfull and create the path if needed
+    directory = os.path.dirname(filename)
+    if not os.path.exists(directory):
+        os.makedirs(directory)
```

Comment:
@boegel of course you can, read my argument again, new Exceptions can be invented in the future.
@stdweird this will hide the original stack-trace, making it harder to pinpoint the problem.
`    raise sys.exc_info()[0], (sys.exc_info()[1],"Cannot create logdirectory %s: %s" % (directory, e), sys.exc_info()[2]` might be an option, but It's not very clean imho.
(see https://docs.python.org/2/reference/simple_stmts.html#the-raise-statement, https://docs.python.org/2/library/sys.html#sys.exc_info )